tutorials/data/combining with syncer example project/group 3/807825fd-5805-4510-b843-8b9ace3090da.json

```diff
@@ -0,0 +1,222 @@
+{
+  "name": "tutorials/data/Combining with syncer example project/Group 3",
```

Comment:
Should have more meaningful name.
Review the port and node names.
Review the data types.